prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml

```diff
@@ -84,6 +84,9 @@ presubmits:
             - --job-config-path=../../knative/test-infra/config/prod/prow/jobs
             - --plugin-config=prow/knative/plugins.yaml
             - --strict
+            # TODO: Re-enable these once we're done experimenting with CODEOWNERS in net-contour.
+            - --exclude-warning=mismatched-tide
```

Comment:
This is already excluded on line 92.

Reply:
I removed this and moved the whole block down.